locales/it_IT/pcsx2_Main.po

```diff
@@ -2942,24 +2933,24 @@ msgid "Clear ISO list"
 msgstr "Cancella lista ISO"
 
 #: pcsx2/gui/Saveslots.h:96
-#, fuzzy, c-format
+#, c-format
 msgid "Slot %d - Empty"
-msgstr "Slot %d"
+msgstr "Slot %d - Vuoto"
 
 #: pcsx2/gui/Saveslots.h:98
-#, fuzzy, c-format
+#, c-format
 msgid "Slot %d - %s %s"
-msgstr "Slot %d"
+msgstr "Slot %d - %s  %s"
 
 #: pcsx2/gui/Saveslots.h:100
 #, c-format
 msgid "Slot %d - Unknown Time"
-msgstr ""
+msgstr "Slot %d - Tempo Sconosciuto"
 
 #: pcsx2/gui/Saveslots.h:113
 #, c-format
 msgid "Write time is %s %s."
-msgstr ""
+msgstr "Tempo scritto è %s %s."
```

Comment:
Tempo scrittura?
(mi sfugge cosa starebbe sta cosa)

Reply:
Sembra che esiste per contare l'ora esatta in cui è stato scritto un file di salvataggio.

Tempo di scrittura è migliore rispetto a come lo avevo tradotto.